chapters/02_hello.md

```diff
@@ -1,65 +1,3 @@
-# About this document 
```

Comment:
I split what was chapter 1 into two chapters, so that Chapter 3 -> 3-Person Patterns, etc.